LWMS ID 5761: Removed expired PARA Links and added alternative texts for all the images in all of the workshop labs

plforacle opened the pull request:
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

arabellayao reviewed (changes requested):
Hi! Image alternative texts should be included in the following format:
![Alt text](./images/image.png " ")
Please let me know if you have any questions. Thanks!

plforacle commented:
Arabella,
I have made the requested changes:
LWMS ID 5761: Changed all Image alternative texts to the following format ![mds heatwave](./images/00_mds_heatwave_2.png "mds heatwave ")

Thank you,
Perside Foster | MySQL Principal Partners Solution Engineer
email: ***@***.******@***.***>
phone: 1.786.877.9063
Oracle  Florida

From: Arabella Yao ***@***.***>
Sent: Tuesday, October 25, 2022 8:33 PM
To: oracle-livelabs/analytics-ai ***@***.***>
Cc: Perside Foster ***@***.***>; Author ***@***.***>
Subject: [External] : Re: [oracle-livelabs/analytics-ai] LWMS ID 5761: Removed expired PARA Links and added alternative texts for all the images in all of the workshop labs (PR #76)


@arabellayao requested changes on this pull request.

Hi! Image alternative texts should be included in the following format:
[Image removed by sender. Alt text]<images/image.png>
Please let me know if you have any questions. Thanks!

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/oracle-livelabs/analytics-ai/pull/76*pullrequestreview-1155705579__;Iw!!ACWV5N9M2RV99hQ!PV1hOpqQHzo8HnhwdadDqTMl8QVJLC7Y-YwKh6Kgmrb4ZUTfi_bhLrj8DcYnx_iUHA1KP79moYVYoqY5K_Eo8qcuVpee$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AMYQBSXSQ52LPAEH6INLCXLWFB335ANCNFSM6AAAAAAROOJCFM__;!!ACWV5N9M2RV99hQ!PV1hOpqQHzo8HnhwdadDqTMl8QVJLC7Y-YwKh6Kgmrb4ZUTfi_bhLrj8DcYnx_iUHA1KP79moYVYoqY5K_Eo8pZUwscV$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
